scripts/load_pickle.py

```diff
@@ -0,0 +1,9 @@
+import pickle
```

Comment:
Do we need this file?

Reply:
May not be necessary, but for checking pickle file content? Would you prefer removing it?

Reply:
Removed, thanks